### ISSUE TITLE
https://gopkg.in/fsnotify.v1no longer exist on this url

### ISSUE THREAD

**Issue by CorentinDy** [unknwon/bra#19]
Hi,
when trying to install your tool to build backend of Grafana,
```go get github.com/Unknwon/bra/``` is trying to download from this url `https://gopkg.in/fsnotify.v1` and it return a `404`.

```
go get -v -u github.com/Unknwon/bra
github.com/Unknwon/bra (download)
github.com/BurntSushi/toml (download)
github.com/Unknwon/com (download)
github.com/Unknwon/log (download)
github.com/urfave/cli (download)
Fetching https://gopkg.in/fsnotify.v1?go-get=1
Parsing meta tags from https://gopkg.in/fsnotify.v1?go-get=1 (status code 404)
package gopkg.in/fsnotify.v1: unrecognized import path "gopkg.in/fsnotify.v1" (parse https://gopkg.in/fsnotify.v1?go-get=1: no go-import meta tags ())
```

Any idea how to fix it ? 

thx

**Comment by CorentinDy** [comment]
https://github.com/go-fsnotify/fsnotify/issues/1

**Comment by zhuharev** [comment]
I'm fix it in #20

**Comment by zhuharev** [comment]
@Unknwon good time for use dep manager :smile: :
